src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -315,3 +382,16 @@
     border: none;
     border-top: 1px solid $bdl-gray-20;
 }
+
+@include breakpoint($small-screen) {
```

Comment:
Could this be moved inside the block above so the `breakpoint` blocks are near each other?

Reply:
Done